crypto/nrf_oberon/include/ocrypto_rsa_key.h

```diff
@@ -78,34 +79,35 @@ typedef struct {
 /**
  * 2048-bit RSA public key.
  */
-/**@cond */
 typedef struct {
+    /**@cond */
     uint32_t n[64];
     // e = 65537
+    /**@endcond */
 } ocrypto_rsa2048_pub_key;
-/**@endcond */
 
 /**
  * 2048-bit RSA secret key.
  */
-/**@cond */
 typedef struct {
+    /**@cond */
     uint32_t n[64];
     uint32_t d[64];  // x^(e*d) mod n == x
+    /**@endcond */
 } ocrypto_rsa2048_key;
-/**@endcond */
 
 /**
  * 2048-bit RSA secret key with CRT coefficients.
  */
-/**@cond */
 typedef struct {
+    /**@cond */
     uint32_t n[64];
     uint32_t p[32], q[32];   // primes, p*q = n
     uint32_t dp[32], dq[32]; // d mod (p-1), d mod (q-1)
     uint32_t qinv[32];       // 1/q mod p
+    /**@endcond */
 } ocrypto_rsa2048_crt_key;
-/**@endcond */
+/**@}*/
```

Comment:
What are all of these for?

Reply:
@cond/@endcond makes it possible to have structs that are not doxygen documentations without warnings